pkgs/applications/misc/udiskie/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, asciidoc-full, gettext
+, gobjectIntrospection, gtk3, hicolor_icon_theme, libnotify
+, pythonPackages, udisks2, wrapGAppsHook }:
+
+pythonPackages.buildPythonApplication rec {
+  name = "udiskie-${version}";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "coldfix";
+    repo = "udiskie";
+    rev = version;
+    sha256 = "01x5fvllb262x6r3547l23z7p6hr7ddz034bkhmj2cqmf83sxwxd";
+  };
+
+  preConfigure = ''
+    export XDG_RUNTIME_DIR=/tmp
```

Comment:
As far as I can tell the `XDG_RUNTIME_DIR` variable doesn't matter at all in the udiskie build. The package builds and runs just fine without it. I removed it and force-pushed.